### PR TITLE
Update the docs to match the new Information Architecture

### LINKED ISSUE #1628: Docs IA update
Based on user feedback and to improve the information architecture, we are creating a "Getting Started" folder. All links associated with the changed files have been modified accordingly.

- [x] Remove operator manual 
- [x] Create "Getting Started" file
- [x] Move Understand the Basics, Core Concepts, and Getting Started into one folder
- [x] Fix any broken links that to pointed set up and install

Relates to #1397 

[PUBLIC - Zarf Project - New Docs Information Architecture.pdf](https://github.com/defenseunicorns/zarf/files/11297901/PUBLIC.-.Zarf.Project.-.New.Docs.Information.Architecture.pdf)

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This is a pretty big restructuring/rewrite of our docs per the IA updates suggested by @Jessy-Morris and @Madeline-UX 

## Related Issue

Fixes #1628 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Other (security config, docs update, etc)

## Checklist before merging

- [X] Test, docs, adr added or updated as needed
- [X] [Contributor Guide Steps](https://github.com/defenseunicorns/zarf/blob/main/CONTRIBUTING.md#developer-workflow) followed

### CODE REVIEW

File: docs/6-zarf-tutorials/7-publish-and-deploy.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ In this tutorial, we are going to run through how to publish a Zarf package to a
 
 For following along locally, please ensure the following prerequisites are met:
 
-1. Zarf binary installed on your `$PATH`: ([Install Instructions](../1-getting-started/index.md#installing-zarf))
-2. Access to a [Registry supporting the OCI Distribution Spec](https://oras.land/implementors/#registries-supporting-oci-artifacts), this tutorial will be using Docker Hub
-3. Access to a cluster that has been [initialized with zarf](./1-initializing-a-k8s-cluster.md).
+- Zarf binary installed on your `$PATH`: ([Install Instructions](../1-getting-started/index.md#installing-zarf))
```
Comment (Noxsios) [inline docs/6-zarf-tutorials/7-publish-and-deploy.md:16]:
```suggestion
- Zarf binary installed on your `$PATH`: ([Installing Zarf](../1-getting-started/index.md#installing-zarf))
```

File: src/pkg/utils/io.go [hunk 2]
```diff
@@ -293,7 +293,7 @@ func IsTextFile(path string) (bool, error) {
 func GetDirSize(path string) (int64, error) {
 	dirSize := int64(0)
 
-	// Walk through all files in the path
+	// Tutorial all files in the path
```
Comment (Noxsios) [inline src/pkg/utils/io.go:296]:
```
	// Tutorial all files in the path
```

lol

File: src/pkg/packager/create.go [hunk 3]
```diff
@@ -474,7 +474,7 @@ func generatePackageChecksums(basePath string) (string, error) {
 	// Add a '/' or '\' to the basePath so that the checksums file lists paths from the perspective of the basePath
 	basePathWithModifier := basePath + string(filepath.Separator)
 
-	// Walk through all files in the package path and calculate their checksums
+	// Tutorial all files in the package path and calculate their checksums
```
Comment (Noxsios) [inline src/pkg/packager/create.go:477]:
```
	// Tutorial all files in the package path and calculate their checksums
```

lol

**Review by Noxsios — CHANGES_REQUESTED** [review]

File: src/pkg/utils/io.go (hunk 2 above)
Comment (Racer159) [reply src/pkg/utils/io.go]: done

File: src/pkg/packager/create.go (hunk 3 above)
Comment (Racer159) [reply src/pkg/packager/create.go]: done

**Pull request merged**